Tasks/Maven/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -6,7 +6,7 @@
   "loc.group.displayName.junitTestResults": "JUnit Test Results",
   "loc.group.displayName.codeCoverage": "Code Coverage",
   "loc.group.displayName.advanced": "Advanced",
-  "loc.group.displayName.SQAnalysis": "SonarQube Analysis",
+  "loc.group.displayName.CodeAnalysis": "Code Analysis",
```

Comment:
Consider giving more context to the PR and how you're breaking up the MMF into smaller PRs, for example it would be helpful to understand what the next planned change brings.